src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPoolIdentity.Unix.cs

```diff
@@ -14,8 +14,11 @@ partial class DbConnectionPoolIdentity
     {
         internal static DbConnectionPoolIdentity GetCurrent()
         {
-            throw new PlatformNotSupportedException();
+            string sidString = System.Environment.UserDomainName + "\\" + System.Environment.UserName;
```

Comment:
Just verifying: is this correct if UserDomainName is empty?  I'm wondering if still having the slash is correct in that case.